kolibri/plugins/setup_wizard/assets/src/state/constants.js

```diff
@@ -0,0 +1,3 @@
+import presets from '../../../../../auth/constants/facility_configuration_presets.json';
```

Comment:
Do you think it'd be a bad idea to have an API endpoint that returns the names of each preset? Importing a file that lives outside of the plugin directly feels kinda weird to me.